cmd/nexd/main.go

```diff
@@ -325,6 +331,12 @@ func main() {
 				EnvVars:  []string{"NEXD_STATE_DIR"},
 				Category: nexServiceOptions,
 			},
+			&cli.StringSliceFlag{
+				Name:     "stun-server",
+				Usage:    "stun server to use discover our endpoint address",
```

Comment:
and adding a note here that we require 2 different stun servers

Reply:
But we won't have two in the e2e tests... or should we spin up two stun servers?